crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -123,8 +116,11 @@ impl Extension for LoggerExtension {
         next: NextExecute<'_>,
     ) -> Response {
         let resp = next.run(ctx, operation_name).await;
+        let session_id = ctx.data_unchecked::<SocketAddr>().to_string();
+        let query_id = ctx.data_unchecked::<Uuid>().to_string();
         if resp.is_err() {
             for err in &resp.errors {
+                let error_code = &err.extensions.as_ref().and_then(|x| x.get("code"));
```

Comment:
I was initially going to suggest a further refinement of this, but I think it's probably not worth it, because the type at this point is not `Option<&str>` it's `Option<Value>`, so it's going to be a pain to create the version of this that contains `UNKNOWN`.